pkg/generate/crossplane/crossplane.go

```diff
@@ -152,13 +152,12 @@ func Crossplane(
 	targets := []string{
```

Comment:
Line 132 should have the full path like:
```
outPath := filepath.Join("apis", metaVars.ServiceIDClean, metaVars.APIVersion, "zz_"+strings.TrimSuffix(filepath.Base(path), ".tpl"))
```

Reply:
sorry. fixed.